wix-blog-backend/wix-blog-backend/DraftPosts/guides/draft-posts-intro.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Introduction
+---
+
+# Introduction
+
+The Draft Posts API gives site members the ability to access and manage draft blog posts.
+A draft post is an unpublished article written for a blog by site collaborators. 
+A blog can have up to 100K draft posts. The size limit of a single draft blog post is 400KB.
+
+To use the Draft Posts API, import `{ draftPosts }` from the `wix-blog-backend` module:
+
+```javascript
+import { draftPosts } from 'wix-blog-backend';
+```
+
+> **Note:** `translationId` will only be returned in the `post` object if the
```

Comment:
This was taken from [Posts](https://www.wix.com/velo/reference/wix-blog-backend/posts/introduction). If I change it here, should I change it everywhere it is used?

Added the conditional statement in REST for Draft Posts, however it will also need to be done for Posts and anywhere else it applies. Should I do it, or open a docbug?